### PR TITLE
[APPS-3459] Update en.yml file with marketplace listing content

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

/cc @zendesk/vegemite, @zendesk/i18n

### Description

Update en.yml file with marketplace listing content

### References
* JIRA: https://zendesk.atlassian.net/browse/APPS-3459

### Risks
* [low] - app content may not show on marketplace

### CODE REVIEW

**Comment by samgavinio** [comment]
Hi :wave @pmgrove, if this is for v1 of shopify_app, this should now be merged into the `v1` branch. `master` now contains the v2 of the app as per https://zendesk.atlassian.net/browse/MI-1644.

Thanks!

**Comment by svizzari** [comment]
@pmgrove This PR needs some attention

**Comment by pmgrove** [comment]
@svizzari: I have updated the base from `master` to `v1` as per @samgavinio's instructions.

File: translations/en.yml [hunk 1]
```diff
@@ -5,10 +5,53 @@ packages:
...
+
+* **Title:** Name for the application.
+
+* **Store URL:** Your Shopify store URL. An example url is https://zendesk-demo.myshopify.com
+
+* **API Key:** An API key generated by Shopify. See the 'Setup' section of the Shopify documentation for [instructions](http://wiki.shopify.com/Private_applications) about how to create an API key for private apps.
+
+* **Password:** Password generated by Shopify, along with the API key.
+
+* Order ID Field ID: The Zendesk Support custom field ID you are using to store the associated Shopify order number. [Click here](https://support.zendesk.com/hc/en-us/articles/203661496) for more information about adding and using custom ticket fields in Zendesk Support.
```
Comment (svizzari) [inline translations/en.yml]: Is " Order ID Field ID" supposed to be bolded too? All the other bullets are

File: translations/en.yml [hunk 2]
```diff
@@ -5,10 +5,53 @@ packages:
...
+      key: "txt.apps.shopify_app.app.installation_instructions"
+      title: "Installation Instructions for App: Shopify"
+      value: "Sign in to your Zendesk Support account, then click the Admin icon in the sidebar. Select Apps > Marketplace, then find the Shopify app and click the tile. On the Shopify app page, click Install app in the upper-right corner. Enter the following information:
+
+
+* **Title:** Name for the application.
+
+* **Store URL:** Your Shopify store URL. An example url is https://zendesk-demo.myshopify.com
+
+* **API Key:** An API key generated by Shopify. See the 'Setup' section of the Shopify documentation for [instructions](http://wiki.shopify.com/Private_applications) about how to create an API key for private apps.
```
Comment (HiroIguchi) [inline translations/en.yml]: This URL redirected me to https://help.shopify.com/api/getting-started/api-credentials#generate-private-app-credentials Is this expected?

Comment (pmgrove) [inline translations/en.yml]: @iandjx, can you confirm that this link is correct?

Comment (iandjx) [inline translations/en.yml]: @HiroIguchi  @pmgrove  This is correct. The user has to create a private app in Shopify to generate an API key

**Comment by mariasica** [comment]
👍

**Pull request merged**